airbase/pom.xml

```diff
@@ -433,6 +430,9 @@
                             -XX:+ExitOnOutOfMemoryError
                             -XX:+HeapDumpOnOutOfMemoryError
                             -XX:-OmitStackTraceInFastThrow
+                            -Duser.timezone=${air.test.timezone}
```

Comment:
I wonder if other properties get's propagated

Reply:
Indeed. Do we need to do the same for java.awt.headless and the java.util.logging.SimpleFormatter property?

Reply:
If i understand this correctly, the difference between `argLine` and `systemPropertyVariables` is that the latter gets applied by some Maven class, from within forked JVM, so after some Java code has already run. This is probably what made setting timezone ineffective -- it's too late.
It might be the case that ` java.awt.headless` doesn't matter until you try to use some graphics classes, but I agree it looks safer to just use `argLine` exclusively.

Reply:
I realized I am wrong. This has nothing to do with `argLine` vs `systemPropertyVariables` -- just presto-main defines `systemPropertyVariables` and overrides `systemPropertyVariables` defined in airbase. Instead of merging into it. I'll try to fix this in Presto instead